src/get_family_names.py

```diff
@@ -0,0 +1,152 @@
+    for name in dict_df['parents_list']:
+        result.append({'id': dict_df['id'],
+                       'relative_name': name,
+                       'relationship': 'son_of'})
+    return pd.DataFrame(data=result)
+
+
+def get_all_congress_people_ids():
+    ids_series = [read_csv(name)['congressperson_id']
+                  for name in ['current-year', 'last-year', 'previous-years']]
+    return list(pd.concat(ids_series).unique())
+
+
+def read_csv(name):
+    return pd.read_csv('data/2016-08-08-%s.xz' % name,
+                       parse_dates=[16],
+                       dtype={'document_id': np.str,
+                              'congressperson_id': np.str,
+                              'congressperson_document': np.str,
+                              'term_id': np.str,
+                              'cnpj_cpf': np.str,
+                              'reimbursement_number': np.str})
+
+
+def write_formatted_data(df):
+    people_with_two_or_less_parents = (
+        df
+        [df.parents_list.apply(len) <= 2]
+        [['id', 'parents_list']])
+
+    final = (
+        people_with_two_or_less_parents
+        .groupby('id')
+        .apply(flatten_parents)
+        .reset_index(drop=True))
+
+    final.rename(inplace=True, columns={'id': 'congressperson_id'})
+    final.to_csv('-'.join(DATE, PROCESSED_DATA_PATH), **write_csv_params)
+
+
+def write_raw_data(df):
+    people_with_more_than_two_parents = (
+        df
+        [df.parents_list.apply(len) > 2]
+        [['id', 'Filiação']])
+
+    people_with_more_than_two_parents.rename(inplace=True, columns={
+        'id': 'congressperson_id',
+        'Filiação': 'parents'})
+
+    people_with_more_than_two_parents.to_csv('-'.join(DATE, RAW_DATA_PATH),
+                                             **write_csv_params)
+
+
+def main():
+    url = 'http://www2.camara.leg.br/deputados/pesquisa/layouts_deputados_' \
+          'biografia?pk={}'
+
+    ids = get_all_congress_people_ids()
+
+    global dicts
+    total = len(ids)
+    for i, id in enumerate(ids):
+        id = str(id).replace('\n', '').strip()
+        print('{} - {:.2f}%'.format(id, i / total * 100))
+        try:
+            data = requests.get(url.format(id)).content.decode('utf8')
+            soup = BeautifulSoup(str(data), 'html.parser')
+            bio_details = soup.findAll('div', {'class': 'bioDetalhes'})
+            contents_bio_details = extract_contents_from_div(bio_details)
+            dict_bio_details = convert_to_dict(contents_bio_details)
+            dict_bio_details['id'] = id
+            dicts.append(dict_bio_details)
+        except IndexError:
+            print('Could not parse data')
+
+    df = pd.DataFrame(data=dicts)
+    df = df[df['Filiação'].notnull()]
+    df = df[~(df['Filiação'].str.contains('Escolaridade'))]
+    df['parents_list'] = df['Filiação'].apply(split_names)
+
+    write_formatted_data(df)
+    write_raw_data(df)
+
+
+dicts = []
```

Comment:
**If** this a matter of facilitating debug, I'd remove line 150 and stop using `global`…